base/multidimensional.jl

```diff
@@ -89,6 +89,7 @@ module IteratorsMD
 
     # indexing
     getindex(index::CartesianIndex, i::Integer) = index.I[i]
+    Base.get(A::AbstractArray, I::CartesianIndex, default) = get(A, I.I, default)
```

Comment:
This is probably not the ideal place for this definition. Not sure, where it should go instead.

Reply:
Yeah, I think it's ok. Ideally it'd go near the rest of the `get` methods, but that won't work due to bootstrapping since they live in `abstractarray.jl`.